services-js/registry-certs/client/store/BirthCertificateRequest.ts

```diff
@@ -22,11 +23,16 @@ export const INITIAL_REQUEST_INFORMATION: Readonly<
   parent2LastName: '',
 };
 
-export const DEFAULT_STEPS: Step[] = [
+export const QUESTION_STEPS: Step[] = [
```

Comment:
I like how you’ve broken down all the steps into their logical groupings here; it really improves readability.